Automattic/wp-calypso
# Stats: Adding empty state component for Referrers

Related to https://github.com/Automattic/red-team/issues/63

## Proposed Changes

* adding new empty state for Referrers
* refactoring empty state actions to be shared across empty states
* extend empty card styling to stretch the main content

## Why are these changes being made?


* a part of the empty states project

## Testing Instructions



* go to the live branch
* apply `stats/empty-module-traffic` feature flag
* verify that a page with empty Referrers component works with and without the feature flag
* repeat for blogs with data

![SCR-20240627-mowk](https://github.com/Automattic/wp-calypso/assets/112354940/c848f01a-d0fd-4118-b12f-9f407ac8e87c)


## Pre-merge Checklist
